### PR TITLE
Many small frontend improvement

### PR DESCRIPTION
**Pull request by vickipetrova**
# Pull Request Template

## Description

Fleshing out the design and UI before we start testing with real users this week. This PR actually grew and became a bit bigger. 

## Link to Ticket

Please indicate the ticket(s) in Notion that correspond to this PR.

- Ticket 1: no relevant ticket

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## What Has Changed

Please describe in bullets what you have changed.

- The bottom bar with the home button is now like a bottom navigation bar: it floats above all other elements and users can still scroll the pages while the nav bar remains at the bottom
- The homepage has a different background
- The room cars have a new design
- The announcements have an improved design
- The top bar has  a settings page
- The settings page leads to an email so users can submit feedback
- Top nav bar also floats
- Settings in the bottom nav bar
- Dark Mode

## How Has This Been Tested / How Can This Be Tested

- Clicked through the app. It's mostly styling changes. 
Screenshot 2024-04-02 at 1 31 32 PM

Screenshot 2024-04-02 at 1 31 46 PM

Screenshot 2024-04-02 at 1 32 02 PM

### CODE REVIEW

File: src/app/_components/TopBar/TopBar.js [hunk 1]
```diff
@@ -20,7 +20,16 @@ export default function TopBar({ title, details, slug }) {
             <p className={styles.header}>{title}</p>
           </>
         )}
+
+        <>
+        <Link href="/app/settings">
+          <div className={styles.settingsContainer}>
+            {/* <FaCog size={22} color={"white"} /> */}
```
Comment (vickipetrova) [inline src/app/_components/TopBar/TopBar.js:27]: This is a settings gear icon to make it look nicer. If I add this, the app crashes so I commented it out. ANyone know how to fix this?

Comment (bathientran) [reply src/app/_components/TopBar/TopBar.js]: I moved the gear to the bottom and added the icon

**Review by bathientran — APPROVED** [review]
Adjusted a couple of things and added my own stuff in here as well

**Comment by vickipetrova** [comment]
I'm not sure if I broke it, but I can't fix this error
image

**Comment by bathientran** [comment]
Should have fixed it – a bit weird because each announcement should correspond to a room member but maybe an edge case that isn't handled -- i guess does the building manager count as a room member?

**Comment by vickipetrova** [comment]
> Should have fixed it – a bit weird because each announcement should correspond to a room member but maybe an edge case that isn't handled -- i guess does the building manager count as a room member?

I think the manager counts too because often the manager can make announcements directly to the rooms - cleaning on Tuesday, new document uploaded, etc. We can merge it now though because we don't really have the manager side of the app anyway so it doesn't matter for the MVP.

**Pull request merged**